### DISCUSSION TITLE
Visually distinct pinned and unpinned tab areas

### DISCUSSION THREAD

**Discussion by Ardemus** [mbnuqw/sidebery#1689] [Ideas]
Turning titles on for pinned tabs makes them blend into my normal tabs. I'd like to set pinned tab background and foreground colors, but there may be better solutions.

**Comment by redrockvp** [comment]
There are some pretty cool things you can do with the styling already. 

Have a look at ArcWTF for instance: https://github.com/kikaraage/arcwtf
